server/middleware/signUpValidation.js

```diff
@@ -39,6 +39,12 @@ const signUp = (req, res, next) => {
       message: 'fullName must contain at least one alphabet',
     });
   }
+  if (req.body.fullName.length < 6) {
+    return res.status(400).send({
+      success: 'false',
+      message: 'Minimum fullName length is 6',
```

Comment:
You could improve on the error message so the user understands what you talking about